examples/schemas/global.json

```diff
@@ -0,0 +1,39 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"type": "object",
+	"definitions": {
+		"time_span": {
+			"type": "string",
+			"pattern": "^[0-9]+:[0-5]?[0-9]$"
+		}
+	},
+	"properties": {
+		"name": {
+			"type": "string"
+		},
+		"staffId": {
+			"type": "integer",
+			"minimum": 1000000,
+			"maximum": 9999999
+		},
+		"department": {
+			"type": "string"
+		},
+		"workingTime": {
+			"$ref": "#/definitions/time_span"
+		},
+		"wage": {
+			"type": "number",
+			"minimum": 0.0
+		},
+		"workingArea": {
+			"type": "string",
+			"enum": ["ub", "gf"]
```

Comment:
Is this enum case sensitive?

Reply:
Yes, it is and to my knowledge there's no way to make the enum case-insensitive. We could simply write all possible values ("ub", "uB", Ub", ..) in the enum, but that makes the autocomplete feature confusing, since all possibilities show up.

Reply:
That's why I chose to go with an enum (and change the example global.json). Would you prefer a different solution @gstuer ?

Reply:
I take your point. Considering that, your implemented solution is fine :)